File: vendor .cs
```csharp
using System;
using System.Collections.Generic;
using Microsoft.Maui;
using Microsoft.Maui.Controls;
using Microsoft.Maui.Graphics;

namespace VendorOnboardingApp
{
    public class vendor : ContentPage
    {
        private bool termsAccepted = false;

        public vendor()
        {
            Title = "Vendor Onboarding";
            BackgroundColor = Colors.White;

            var organisationSection = CreateAccordionSection("Organisation", true, CreateOrganisationItems());
            var factoryDetailsSection = CreateAccordionSection("Factory Details", false, null);
            var productCapacitySection = CreateAccordionSection("Product Capacity", false, null);
            var financialInfoSection = CreateAccordionSection("Financial Info", false, null);

            var termsCheckbox = new CheckBox
            {
                Color = Color.FromArgb("#e25532"),
                IsChecked = termsAccepted
            };
            termsCheckbox.CheckedChanged += (s, e) => termsAccepted = termsCheckbox.IsChecked;

            var termsLabel = new Label
            {
                Text = "Terms & Conditions",
                FontSize = 16,
                TextColor = Color.FromArgb("#666666"),
                VerticalOptions = LayoutOptions.Center,
                Margin = new Thickness(10, 0, 0, 0)
            };

            var termsStack = new StackLayout
            {
                Orientation = StackOrientation.Horizontal,
                HorizontalOptions = LayoutOptions.Center,
                Margin = new Thickness(0, 20, 0, 20),
                Children = { termsCheckbox, termsLabel }
            };

            var submitButton = new Button
            {
                Text = "Submit for Review",
                BackgroundColor = Color.FromArgb("#e25532"),
                TextColor = Colors.White,
                FontAttributes = FontAttributes.Bold,
                CornerRadius = 5,
                Margin = new Thickness(20, 0, 20, 20),
                HeightRequest = 50
            };

            Content = new StackLayout
            {
                Children = {
                    organisationSection,
                    factoryDetailsSection,
                    productCapacitySection,
                    financialInfoSection,
                    termsStack,
                    submitButton
                }
            };

            // Add header toolbar with hamburger menu
            NavigationPage.SetHasNavigationBar(this, false);
            
            var menuIcon = new Image
            {
                Source = "hamburger_menu.png",
                HeightRequest = 24,
                WidthRequest = 24
            };

            var menuButton = new ImageButton
            {
                Source = "hamburger_menu.png",
                HeightRequest = 24,
                WidthRequest = 24,
                BackgroundColor = Colors.Transparent,
                Padding = new Thickness(10)
            };

            var titleLabel = new Label
            {
                Text = "Vendor Onboarding",
                TextColor = Colors.White,
                FontSize = 20,
                FontAttributes = FontAttributes.Bold,
                HorizontalOptions = LayoutOptions.Center,
                VerticalOptions = LayoutOptions.Center
            };

            var toolbarContent = new Grid
            {
                ColumnDefinitions =
                {
                    new ColumnDefinition { Width = GridLength.Auto },
                    new ColumnDefinition { Width = GridLength.Star },
                    new ColumnDefinition { Width = GridLength.Auto }
                },
                BackgroundColor = Color.FromArgb("#e25532"),
                HeightRequest = 60,
                Padding = new Thickness(10, 0)
            };

            toolbarContent.Add(menuButton, 0, 0);
            toolbarContent.Add(titleLabel, 1, 0);

            // Add the toolbar to the top of the content
            var mainLayout = new Grid
            {
                RowDefinitions =
                {
                    new RowDefinition { Height = GridLength.Auto },
                    new RowDefinition { Height = GridLength.Star }
                }
            };

            mainLayout.Add(toolbarContent, 0, 0);
            
            var contentScrollView = new ScrollView
            {
                Content = Content
            };
            
            mainLayout.Add(contentScrollView, 0, 1);
            
            Content = mainLayout;
        }

        private Frame CreateAccordionSection(string title, bool isExpanded, StackLayout content)
        {
            var headerLabel = new Label
            {
                Text = title,
                FontSize = 18,
                TextColor = Color.FromArgb("#666666"),
                VerticalOptions = LayoutOptions.Center
            };

            var chevronImage = new Image
            {
                Source = isExpanded ? "chevron_up.png" : "chevron_down.png",
                HeightRequest = 24,
                WidthRequest = 24,
                HorizontalOptions = LayoutOptions.End,
                VerticalOptions = LayoutOptions.Center
            };

            var headerGrid = new Grid
            {
                ColumnDefinitions =
                {
                    new ColumnDefinition { Width = GridLength.Star },
                    new ColumnDefinition { Width = GridLength.Auto }
                },
                Padding = new Thickness(15, 10)
            };

            headerGrid.Add(headerLabel, 0, 0);
            headerGrid.Add(chevronImage, 1, 0);

            var headerTapGesture = new TapGestureRecognizer();
            headerGrid.GestureRecognizers.Add(headerTapGesture);

            var mainStack = new StackLayout();
            mainStack.Add(headerGrid);
            
            if (isExpanded && content != null)
            {
                mainStack.Add(content);
            }

            var frame = new Frame
            {
                Content = mainStack,
                Padding = new Thickness(0),
                Margin = new Thickness(20, 10),
                CornerRadius = 10,
                HasShadow = false,
                BorderColor = Color.FromArgb("#EEEEEE")
            };

            headerTapGesture.Tapped += (s, e) =>
            {
                isExpanded = !isExpanded;
                chevronImage.Source = isExpanded ? "chevron_up.png" : "chevron_down.png";
                
                if (isExpanded && content != null)
                {
                    if (!mainStack.Children.Contains(content))
                    {
                        mainStack.Add(content);
                    }
                }
                else
                {
                    if (content != null && mainStack.Children.Contains(content))
                    {
                        mainStack.Remove(content);
                    }
                }
            };

            return frame;
        }

        private StackLayout CreateOrganisationItems()
        {
            var stack = new StackLayout();

            // Create the list items
            var items = new List<(string Title, string IconName, bool IsCompleted, bool HasChevron)>
            {
                ("Primary Info", "info_circle", true, false),
                ("Contact Info", "info_circle", false, true),
                ("Workplace & Workforce Info", "info_circle", false, true),
                ("Quality Assurance Info", "info_circle", false, true),
                ("Bank Details", "info_circle", false, true),
                ("Sub Vendors Details", "info_circle", false, true),
                ("Additional Information", "info_circle", false, true)
            };

            foreach (var item in items)
            {
                stack.Add(CreateListItem(item.Title, item.IconName, item.IsCompleted, item.HasChevron));
            }

            return stack;
        }

        private Frame CreateListItem(string title, string iconName, bool completed, bool hasChevron)
        {
            Color circleColor = completed ? Color.FromArgb("#66BB6A") : Color.FromArgb("#F8D0C2");
            string iconText = completed ? "✓" : "i";

            var iconLabel = new Label
            {
                Text = iconText,
                TextColor = completed ? Colors.White : Color.FromArgb("#e25532"),
                HorizontalOptions = LayoutOptions.Center,
                VerticalOptions = LayoutOptions.Center,
                FontSize = 14,
                FontAttributes = FontAttributes.Bold
            };

            var iconCircle = new Frame
            {
                Content = iconLabel,
                WidthRequest = 30,
                HeightRequest = 30,
                CornerRadius = 15,
                Padding = new Thickness(0),
                HasShadow = false,
                BackgroundColor = circleColor,
                HorizontalOptions = LayoutOptions.Start,
                VerticalOptions = LayoutOptions.Center
            };

            var titleLabel = new Label
            {
                Text = title,
                TextColor = Color.FromArgb("#666666"),
                FontSize = 16,
                VerticalOptions = LayoutOptions.Center,
                Margin = new Thickness(10, 0, 0, 0)
            };

            var chevron = hasChevron ? new Image
            {
                Source = "chevron_right.png",
                HeightRequest = 20,
                WidthRequest = 20,
                HorizontalOptions = LayoutOptions.End,
                VerticalOptions = LayoutOptions.Center
            } : null;

            var grid = new Grid
            {
                ColumnDefinitions =
                {
                    new ColumnDefinition { Width = GridLength.Auto },
                    new ColumnDefinition { Width = GridLength.Star },
                    new ColumnDefinition { Width = GridLength.Auto }
                },
                Padding = new Thickness(15, 10)
            };

            grid.Add(iconCircle, 0, 0);
            grid.Add(titleLabel, 1, 0);
            if (chevron != null)
            {
                grid.Add(chevron, 2, 0);
            }

            var tapGesture = new TapGestureRecognizer();
            grid.GestureRecognizers.Add(tapGesture);

            var separator = new BoxView
            {
                HeightRequest = 1,
                Color = Color.FromArgb("#EEEEEE"),
                HorizontalOptions = LayoutOptions.FillAndExpand
            };

            var stack = new StackLayout
            {
                Children = { grid, separator }
            };

            return new Frame
            {
                Content = stack,
                Padding = new Thickness(0),
                CornerRadius = 0,
                HasShadow = false,
                BorderColor = Colors.Transparent
            };
        }
    }
}
```

File: vendor .swift
```swift
import SwiftUI

struct vendorView: View {
    @State private var organisationExpanded = true
    @State private var factoryDetailsExpanded = false
    @State private var productCapacityExpanded = false
    @State private var financialInfoExpanded = false
    @State private var termsAccepted = true
    
    var body: some View {
        VStack(spacing: 0) {
            // Header
            HStack {
                Button(action: {}) {
                    Image(systemName: "line.horizontal.3")
                        .font(.title2)
                        .foregroundColor(.white)
                }
                
                Spacer()
                
                Text("Vendor Onboarding")
                    .font(.title3)
                    .fontWeight(.medium)
                    .foregroundColor(.white)
                
                Spacer()
            }
            .padding()
            .background(Color(UIColor(red: 0.91, green: 0.30, blue: 0.24, alpha: 1.0)))
            
            ScrollView {
                VStack(spacing: 16) {
                    // Organisation Section
                    VStack(spacing: 0) {
                        HStack {
                            Text("Organisation")
                                .font(.headline)
                                .foregroundColor(.gray)
                            
                            Spacer()
                            
                            Button(action: {
                                organisationExpanded.toggle()
                            }) {
                                Image(systemName: organisationExpanded ? "chevron.up" : "chevron.down")
                                    .foregroundColor(.gray)
                            }
                        }
                        .padding()
                        .background(Color(.systemGray6))
                        
                        if organisationExpanded {
                            VStack(spacing: 0) {
                                // Primary Info - Completed
                                HStack {
                                    ZStack {
                                        Circle()
                                            .fill(Color.green)
                                            .frame(width: 28, height: 28)
                                        
                                        Image(systemName: "checkmark")
                                            .font(.system(size: 14))
                                            .foregroundColor(.white)
                                    }
                                    
                                    Text("Primary Info")
                                        .font(.body)
                                        .padding(.leading, 8)
                                    
                                    Spacer()
                                }
                                .padding(.vertical, 12)
                                .padding(.horizontal)
                                Divider()
                                
                                // Other info items - Pending
                                ForEach(["Contact Info", "Workplace & Workforce Info", "Quality Assurance Info", "Bank Details", "Sub Vendors Details", "Additional Information"], id: \.self) { item in
                                    NavigationLink(destination: EmptyView()) {
                                        HStack {
                                            ZStack {
                                                Circle()
                                                    .fill(Color(UIColor(red: 0.98, green: 0.80, blue: 0.73, alpha: 1.0)))
                                                    .frame(width: 28, height: 28)
                                                
                                                Text("i")
                                                    .font(.system(size: 16))
                                                    .foregroundColor(.gray)
                                            }
                                            
                                            Text(item)
                                                .font(.body)
                                                .foregroundColor(.black)
                                                .padding(.leading, 8)
                                            
                                            Spacer()
                                            
                                            Image(systemName: "chevron.right")
                                                .foregroundColor(.gray)
                                        }
                                        .padding(.vertical, 12)
                                        .padding(.horizontal)
                                    }
                                    .buttonStyle(PlainButtonStyle())
                                    
                                    if item != "Additional Information" {
                                        Divider()
                                            .padding(.leading, 56)
                                    }
                                }
                            }
                            .background(Color.white)
                        }
                    }
                    .background(Color.white)
                    .cornerRadius(10)
                    .shadow(color: Color.black.opacity(0.05), radius: 2)
                    
                    // Factory Details Section
                    ExpandableSectionView(
                        title: "Factory Details", 
                        isExpanded: $factoryDetailsExpanded, 
                        content: {
                            // This would contain factory details content
                            EmptyView()
                        }
                    )
                    
                    // Product Capacity Section
                    ExpandableSectionView(
                        title: "Product Capacity",
                        isExpanded: $productCapacityExpanded,
                        content: {
                            // This would contain product capacity content
                            EmptyView()
                        }
                    )
                    
                    // Financial Info Section
                    ExpandableSectionView(
                        title: "Financial Info",
                        isExpanded: $financialInfoExpanded,
                        content: {
                            // This would contain financial info content
                            EmptyView()
                        }
                    )
                    
                    // Terms & Conditions
                    HStack {
                        Button(action: {
                            termsAccepted.toggle()
                        }) {
                            ZStack {
                                RoundedRectangle(cornerRadius: 4)
                                    .fill(termsAccepted ? Color(UIColor(red: 0.91, green: 0.30, blue: 0.24, alpha: 1.0)) : Color.white)
                                    .frame(width: 24, height: 24)
                                    .overlay(
                                        RoundedRectangle(cornerRadius: 4)
                                            .stroke(termsAccepted ? Color(UIColor(red: 0.91, green: 0.30, blue: 0.24, alpha: 1.0)) : Color.gray, lineWidth: 1)
                                    )
                                
                                if termsAccepted {
                                    Image(systemName: "checkmark")
                                        .font(.system(size: 14))
                                        .foregroundColor(.white)
                                }
                            }
                        }
                        .buttonStyle(PlainButtonStyle())
                        
                        Text("Terms & Conditions")
                            .font(.body)
                            .foregroundColor(.gray)
                            .padding(.leading, 8)
                        
                        Spacer()
                    }
                    .padding()
                    
                    // Submit Button
                    Button(action: {}) {
                        Text("Submit for Review")
                            .font(.headline)
                            .foregroundColor(.white)
                            .frame(maxWidth: .infinity)
                            .padding()
                            .background(Color(UIColor(red: 0.91, green: 0.30, blue: 0.24, alpha: 1.0)))
                            .cornerRadius(8)
                    }
                    .padding(.horizontal)
                    .padding(.bottom, 16)
                }
                .padding(.vertical)
            }
            .background(Color(.systemGray6))
        }
        .edgesIgnoringSafeArea(.top)
    }
}

struct ExpandableSectionView<Content: View>: View {
    var title: String
    @Binding var isExpanded: Bool
    var content: () -> Content
    
    var body: some View {
        VStack(spacing: 0) {
            HStack {
                Text(title)
                    .font(.headline)
                    .foregroundColor(.gray)
                
                Spacer()
                
                Button(action: {
                    isExpanded.toggle()
                }) {
                    Image(systemName: isExpanded ? "chevron.up" : "chevron.down")
                        .foregroundColor(.gray)
                }
            }
            .padding()
            .background(Color(.systemGray6))
            
            if isExpanded {
                content()
                    .padding()
                    .background(Color.white)
            }
        }
        .background(Color.white)
        .cornerRadius(10)
        .shadow(color: Color.black.opacity(0.05), radius: 2)
    }
}

struct vendorView_Previews: PreviewProvider {
    static var previews: some View {
        vendorView()
    }
}
```